### ISSUE TITLE
Proposal to make tags page a word cloud

### ISSUE THREAD

**Issue by 0x4c47** [seanlane/gochowdown#26]
I found a very cool Hugo template to render a word cloud of the used tags with logarithmic distribution of font sizes: https://www.sidorenko.io/post/2017/07/nice-tagcloud-with-hugo/

I implemented this for my Gochowdown site and like/use the tags page a lot more now. It's much more useful than a long list of words imho. This is what it looks like on my site:
![Screenshot from 2021-03-14 20-35-40](https://user-images.githubusercontent.com/43940137/111081700-e26ba180-8504-11eb-91ff-4ba6372ed555.png)

I like it so much that I wanted to suggest making it the default for Gochowdown. The only thing that would need to be customized to include it in the theme is the smallest and biggest font size as far as I can see.

**Comment by seanlane** [comment]
Yeah, seems like a great idea. I'll leave this as an open issue with a 'Help Wanted' label for now, since I don't have time at the moment to implement it.

**Comment by Tiliavir** [comment]
Hi @0x4c47 

I love that! Could you create a PR for that, if you have already implemented it?

Thanks and best regards,
Markus

**Comment by 0x4c47** [comment]
Unfortunately I don't have any experience with Hugo and hacked it together real quick to make it work. I don't know how to correctly embed it into the existing theme.
But I linked the source for the code in the initial description if anyone with more knowledge about this project wants to include it.
